### ISSUE TITLE
Flow “bedrijfsmodes” with “warmwater”

### ISSUE THREAD

**Issue by MikeymRx** [robertklep/nl.nefit.easy#16]
When I make a flow with:
IF ”bedrijfsmodes” changed
AND “bedrijfsmodes” is “warmwater”
THAN turn red light.

This is not working, or am I doing something wrong?

**Comment by robertklep** [comment]
No, you're not doing anything wrong, I confirmed that it's a bug. I will fix it and push a new version of the app tonight (may take a few days before Athom will approve the update).

**Comment by robertklep** [comment]
Try and see if the beta (v3.3.2) fixes your problem.

**Comment by MikeymRx** [comment]
I think you fix the problem!
I’m very happy with It

WHEN… 
[Bedrijfsmodes] is change.
AND… 
[Bedrijfsmodes] is [Warmwater]
THEN… 
turn on [WTW controller]
turn off [WTW controller] after 1200 sec.

I only need a timer before [Bedrijfsmodes] is [Warmwater] for 5-10 minutes.
I can’t fix that, maybe you have a solution?

Thx!
Michael

Van: Robert Klep
Verzonden: maandag 14 oktober 2019 19:36
Aan: robertklep/nl.nefit.easy
CC: MikeymRx; Author
Onderwerp: Re: [robertklep/nl.nefit.easy] Flow “bedrijfsmodes” with “warmwater” (#16)

Try and see if the beta (v3.3.2) fixes your problem.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.



-- 
Dit e-mailbericht is gecontroleerd op virussen met Avast antivirussoftware.
https://www.avast.com/antivirus

**Comment by robertklep** [comment]
> I only need a timer before [Bedrijfsmodes] is [Warmwater] for 5-10 minutes.
> I can’t fix that, maybe you have a solution?

Do you mean that when "bedrijfsmodus" has been "warmwater" for 5 to 10 minutes, you want to turn it off?

**Comment by MikeymRx** [comment]
After detection (warmwater), the detection (warmwater) must be active for 4 to 5 minutes.  because of this, I know that you use the shower.
A normal tap is normally not used for longer than 4-5 minutes. This is the trigger to see if you are under the shower so that the fan comes on for 20 minutes.

**Comment by MikeymRx** [comment]
I fix the problem. I just make a flow that detects “warmwater” and start a second flow with 240 delay. A second flow start the flow and detects again “warmwater” and than start the fan for 20 minutes.

**Comment by MikeymRx** [comment]
2nd problem

Bedrijfsmodes Uit

I don't think it works

**Comment by MikeymRx** [comment]
This is te FLOW that checks if you use hot water (warmwater) for a longer time than 4 minutes.

FLOW1 (1st hot water check)
WHEN
[Bedrijfsmodes] is change
AND
[Bedrijfsmodes] is [Warmwater]
THAN
Start [FLOW2] after 240 sec.

FLOW2 (after 4 minutes 2nd check)
WHEN
[FLOW2] Start
AND

[Bedrijfsmodes] is [Warmwater]

Here fails the FLOW!
The flow continues while hot water (Bedrijfsmodes - Warmwater) is no longer tapped and must be off (Bedrijfsmodes - Uit). It seems if the change in the FLOW not being continued, but in the operation (manuel) it will be change.

THAN
Turn on [Fan]
Turn off [Fan] after 1200 sec.

**Comment by robertklep** [comment]
In FLOW2, what you could try is to remove the condition (AND) card and create a notification that will show the current "bedrijfsmodus" (from the tag).

Something like this:
![image](https://user-images.githubusercontent.com/1190916/67033462-19723380-f116-11e9-974f-9ab8c2d60f36.png)

Just to check what the app thinks the state is. By the way, it may take a while (default is 60 seconds) for the app to update the status, so if you turn off the warm water it will take 60 seconds before the app knows you turned it off. You can change this interval in the app settings, but the minimum update interval is still 30 seconds.

**Comment by MikeymRx** [comment]
Strange, the flow stays on, but “hot water” is no longer detected.
And after the “last check” the fan is going on.

![Image](https://i.imgur.com/HLURxil.jpg)

**Comment by MikeymRx** [comment]
This works when I use the “Logica” option from Homey.
I think there is a problem with the translation “hot water” and “warmwater”.

The flow did not recognize the word. This does not change anything I think, the flow just goes through the controls (1-3 in 3 minutes)

![Image](https://i.imgur.com/ezbOgIU.jpg)

**Comment by robertklep** [comment]
Ah I think there might be an issue with the translated and the _actual_ (internal) values for the condition card :(

**Comment by MikeymRx** [comment]
Until now I am very happy about your app.
I hope I have been able to help you find a solution.
Until now we can be happy with these developers!

Thanks for your quick response and solutions so far.
